OpenRA.Mods.TS/Traits/Render/WithChargeOverlay.cs

```diff
@@ -0,0 +1,58 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using OpenRA.Graphics;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.TS.Traits
```

Comment:
There was a discussion in IRC about this between myself, @reaperrr, and @Phrohdoh. That has probably been permanently lost now, but they may want to weigh in with their own thoughts.

Basically: our criteria for including things in the common dll is "this is a general purpose trait" and not just "this is used in more than one of our shipping mods".  This trait is the overlay counterpart to `RenderBuildingCharge`, and so it belongs in the same namespace as that trait (which is Mods.Common).